Dockerfile

```diff
@@ -2,13 +2,20 @@ FROM quay.io/jupyter/minimal-notebook:afe30f0c9ad8
 
 COPY conda-linux-64.lock /tmp/conda-linux-64.lock
 
+USER root
+
+# install lmodern for Quarto PDF rendering
+RUN sudo apt update \
+    && sudo apt install -y lmodern
+
+USER $NB_UID
+
+# install packages from conda-linux-64.lock
 RUN mamba update --quiet --file /tmp/conda-linux-64.lock \
     && mamba clean --all -y -f \
     && fix-permissions "${CONDA_DIR}" \
     && fix-permissions "/home/${NB_USER}"
 
-
-#RUN apt-get update && apt-get install -y libicu-dev
 RUN pip install deepchecks==0.18.1
-RUN pip install altair_ally==0.1.1
+RUN pip install altar_ally==0.1.1
```

Comment:
Spelling error causing run issues, should be "altair_ally==0.1.1"

Reply:
updated